cmd/jujud/agent/model/manifolds.go

```diff
@@ -96,18 +99,21 @@ type ManifoldsConfig struct {
 	// (typically environs.New).
 	NewEnvironFunc environs.NewEnvironFunc
 
+	// NewBrokerFunc is a function opens a CAAS provider.
+	NewBrokerFunc jujucaas.NewBrokerFunc
```

Comment:
please call the var NewContainerBrokerFunc, to give a bit more context

Reply:
yup